config/run.tf

```diff
@@ -55,7 +56,8 @@ resource "google_cloud_run_service" "gcs_to_bq_service" {
 
         resources {
           limits = {
-            memory = "4G"
+            memory = "8Gi"
```

Comment:
i assume it's fine to use `G` or `Gi` for these units?

Reply:
Yeah its fine